torch/export/_trace.py

```diff
@@ -1279,11 +1287,21 @@ def _strict_export_lower_to_aten_ir(
 
     _normalize_nn_module_stack(gm_torch_level, type(mod))
 
-    # NOTE: graph module expects only positional args
     constant_attrs = _gather_constant_attrs(mod)
+    param_buffer_table: Dict[str, str] = _get_param_buffer_mapping(mod, gm_torch_level)
+
+    # Dynamo does not track which buffers were registered as non-persistent. This info
```

Comment:
How about turn L1291-1300 into a standalone function something like _set_non_persistent_buffers or something?

Reply:
I considered doing that, but `param_buffer_table` is also used later for restoring names.

So if we have to name this function, it feels like another kind of `restore` function. But above we explicitly decided not to restore names in the graph module for some reason (see `restore_fqn` argument) so the whole workflow feels like a hodge-podge. I think the good thing to do long-term would be to understand why we don't do all the restoration in one place.